spring-cloud-stream-app-starters-docs/src/main/asciidoc/overview.adoc

```diff
@@ -492,4 +494,146 @@ Then do this:
 
 This would upload the docker image to the docker hub in your custom repository.
 
+=== General FAQ on Spring Cloud Stream App Starters
+
+In the following sections, you can find a brief faq on various things that we discussed above and a few other infrastructure related topics.
+
+
+. What are Spring Cloud Stream Application Starters? +
+Spring Cloud Stream App Starters are individual repositories hosted under the GitHub organization - spring-cloud-stream-app-starters - on github (http://github.com/spring-cloud-stream-app-starters) +
```

Comment:
This description doesn't seem that good, saying the are repositories.  The description from the project page seems better  to repeat here ```Spring Cloud Stream Application Starters are Spring Boot based Spring Integration applications that provide integration with external systems.```